lectures/long_run_growth.md

```diff
@@ -518,19 +504,17 @@ BEM = ['GBR', 'IND', 'AUS', 'NZL', 'CAN', 'ZAF']
 gdp['BEM'] = gdp[BEM].loc[start_year-1:end_year].interpolate(method='index').sum(axis=1) # Interpolate incomplete time-series
 ```
 
-+++ {"user_expressions": []}
 
 Let's take a look at the aggregation that represents the British Empire.
 
 ```{code-cell} ipython3
-gdp['BEM'].plot() # The first year is np.nan due to interpolation
+gdp['BEM'].plot(ylabel="International $'s") # The first year is np.nan due to interpolation
```

Comment:
This line prints out the label
![Screenshot](https://github.com/QuantEcon/lecture-python-intro/assets/39026988/110dabe1-f905-4a4a-bf1c-37537b10c09c)

We can suppress printing using
```suggestion
_ = gdp['BEM'].plot(ylabel="International $'s") # The first year is np.nan due to interpolation
```

or 

```suggestion
gdp['BEM'].plot(ylabel="International $'s") # The first year is np.nan due to interpolation
plt.show()
```